samples/cellular/location/overlay-nrf700x-wifi-scan-only.conf

```diff
@@ -58,5 +58,6 @@ CONFIG_NET_MGMT_EVENT_STACK_SIZE=1024
 # Wi-Fi shell sample has 25000 heap size. We need a bit more for location request purposes.
```

Comment:
Adding a comment here, because it can not be added on line 23. Please change `CONFIG_HEAP_MEM_POOL_SIZE` to `CONFIG_NRF_WIFI_CTRL_HEAP_SIZE`.